tendrl/commons/objects/job/__init__.py

```diff
@@ -33,17 +33,6 @@ def save(self, ttl=None):
             _parent.children = list(set(_children))
             _parent.save()
 
-        if self.status == "failed":
```

Comment:
This will affect all jobs/flows other than "import_cluster", what about those jobs and their failures?